app/controllers/api/account.php

```diff
@@ -3575,8 +3575,8 @@
     ->label('audits.userId', '{response.$id}')
     ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_JWT])
     ->label('sdk.namespace', 'account')
-    ->label('sdk.method', 'addAuthenticator')
-    ->label('sdk.description', '/docs/references/account/add-authenticator.md')
+    ->label('sdk.method', 'createMfaAuthenticator')
```

Comment:
```suggestion
    ->label('sdk.method', 'createMfaAuthenticators')
```

Reply:
Are you sure? In documents we have `createDocument` even tho endpoint is `/documents`

Reply:
From discord chat, Eldad meant list documents. Note for myself to check all list documents for MFA if they arent singular.

Reply:
All good, double-checked all list endpoints in user& account contorller